### PR TITLE
doc: Fix typos

### PR DESCRIPTION
**Pull request by acktsap**
Fix typos.

### CODE REVIEW

File: src/wallet/test/wallet_tests.cpp [hunk 1]
```diff
@@ -426,7 +426,7 @@ BOOST_AUTO_TEST_CASE(LoadReceiveRequests)
 
 // Test some watch-only LegacyScriptPubKeyMan methods by the procedure of loading (LoadWatchOnly),
 // checking (HaveWatchOnly), getting (GetWatchPubKey) and removing (RemoveWatchOnly) a
-// given PubKey, resp. its corresponding P2PK Script. Results of the the impact on
+// given PubKey, resp. its corresponding P2PK Script. Results of the impact on
```
Comment (jonatack) [inline src/wallet/test/wallet_tests.cpp:429]: I guess someone really likes the band? anyway, ACK but could you also fix this one while here

```
$ test/lint/lint-spelling.sh 
src/crypto/sha256_arm_shani.cpp:65: implemenation ==> implementation
```

Comment (acktsap) [reply src/wallet/test/wallet_tests.cpp:429]: Done! https://github.com/bitcoin/bitcoin/pull/24359/commits/9c2bb5a2b499663c386aa9641baa85558aaa25ff

**Comment by jonatack** [comment]
Can you please squash the commits down to one prefixed by "doc:"?

**Comment by acktsap** [comment]
Squash & force pushed in https://github.com/bitcoin/bitcoin/commit/ba4906f951d9f436d9669701fdb3b6e2884121f3

**Review by ghost — APPROVED** [review]
ACK https://github.com/bitcoin/bitcoin/pull/24359/commits/ba4906f951d9f436d9669701fdb3b6e2884121f3

**Review by Zero-1729 — COMMENTED** [review]
crACK ba4906f951d9f436d9669701fdb3b6e2884121f3

**Review by jonatack — APPROVED** [review]
This is the day your life will surely change https://www.youtube.com/watch?v=7ZYgKCbFbWY

ACK ba4906f951d9f436d9669701fdb3b6e2884121f3

**Pull request merged**